src/observability/summary.go

```diff
@@ -11,7 +12,8 @@ func GetSummaryData(request *opb.Request) (*opb.Response, error) {
 	resp := opb.Response{}
 	var err error = nil
 
-	if request.Type == "system" || request.Type == "all" {
+	if strings.Contains(request.Type, "process") || strings.Contains(request.Type, "file") || strings.Contains(request.Type, "network") {
```

Comment:
can the request.Type be set to "all"? Then this check won't match, right?

Reply:
No @Ankurk99 
From karmor summary, I am planning to pass the request type as 
**"process,file,network"**

Reply:
As we discussed, the default type will be all, i.e "process,file,network" so the user won't need to pass "all".